src/services/namex.services.ts

```diff
@@ -75,7 +75,7 @@ export default class NamexServices {
 
       // if there is no previous request_action message then
       // we just preserve whatever text there is and append msg
-      data['additionalInfo'] += ` \n\n ${msg}`
+      data['additionalInfo'] += `\n\n${msg}`
```

Comment:
There is other code that adds messages to the additional info, with double carriage returns, but without extra spaces. So I removed them here to be consistent.